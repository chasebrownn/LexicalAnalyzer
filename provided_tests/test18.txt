OwIpt (((v)|((U)*)).(V))|(k) #
" vVvV vVUUUUk vVk "