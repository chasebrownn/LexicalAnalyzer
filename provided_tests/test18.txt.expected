OwIpt , "vV"
OwIpt , "vV"
OwIpt , "vV"
ERROR
